### ISSUE TITLE
Upgrade gradle version to 7.x.x

### ISSUE THREAD

**Issue by hieuwu** [hieuwu/android-groceries-store#18]

**Comment by Raghu1124** [comment]
Hey there! I would like to give a try to upgrade the gradle version, please assign me this issue.

**Comment by hieuwu** [comment]
> Hey there! I am a kotlin practicer and would like to give a try to upgrade the gradle version, please assign me this issue. It will be a good experience for me.

This is an old issue. Could you please update to latest version?

**Comment by Raghu1124** [comment]
> > Hey there! I am a kotlin practicer and would like to give a try to upgrade the gradle version, please assign me this issue. It will be a good experience for me.
> 
> This is an old issue. Could you please update to latest version?

Yes of course, I will give my best for it, please assign me this issue. Thanks

**Comment by utkarsh006** [comment]
@hieuwu Hey it's been a week on this issue? Can you please check the updates with him?

**Comment by hieuwu** [comment]
> @hieuwu Hey it's been a week on this issue? Can you please check the updates with him? Else you can assign this issue to me

There is another PR merged for this. But you can continue with improvement

**Comment by utkarsh006** [comment]
@hieuwu Yes sure, thanks Hieu :)
